src/controller/userRouter.js

```diff
@@ -25,7 +25,7 @@ userRouter.post("/login", validateUsername, validatePassword, async (req, res) =
     const password = req.body.password;
 
     try {
-        const token = await login(username, password);
+        const token = await userService.login(username, password);
```

Comment:
I think it would be easier if we imported userService as I did, so all we have to do is use dot notation to use a function instead of having to add our functions to the deconstructor.

Reply:
sounds good. just make sure to update the other functions as well